<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWidget</class>
 <widget class="QWidget" name="CreateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>745</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>createWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <widget class="CreateWizard" name="createWizard"/>
     <widget class="CreatorTypeWidget" name="creatorTypeWidget"/>
     <widget class="AlgoTopoView" name="algoTopoView"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CreateWizard</class>
   <extends>QWidget</extends>
   <header>..create.createWizard</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AlgoTopoView</class>
   <extends>QWidget</extends>
   <header>..create.algoTopoView</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CreatorTypeWidget</class>
   <extends>QWidget</extends>
   <header>..create.creatorTypeWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
